pvnet_app/consts.py

```diff
@@ -1,3 +1,9 @@
 sat_path = "sat.zarr"
 nwp_ukv_path = "nwp_ukv.zarr"
 nwp_ecmwf_path = "nwp_ecmwf.zarr"
+
+
+uk_box = dict(
```

Comment:
Is this used anywhere?

Reply:
ah good point, i can remove this now

Reply:
Okay cool. It looks like in live we don't actually serve the entire area of the EUMETSAT image - but just a cut out centred on the UK. So I think there actually shouldn't be NaNs since we don't have any part of the image which is off the globe